frontend/src/utils/DeviceManager/deviceManager.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Device manager that retrieves the stored device ID for a given device type (audio or video)
```

Comment:
Pedantic: a device manager allows users to manage a device, like adding, removing, or editing it.  This seems more like a `DeviceStore`.

I'll shut up now 🙊

Reply:
sure, actually that's a great suggestion! I was struggling to pick the right name for this and `DeviceManager` sounded good to me but I like `DeviceStore` better